reference/UiTPAS.v2.json

```diff
@@ -1151,7 +1182,182 @@
             "description": "Include only passholders with a dateOfBirth of this value (including) or older. Format yyyy-mm-dd e.g. 2003-01-01"
           }
         ]
-      }
+      },
+      "parameters": []
+    },
+    "/passholders/me": {
+      "get": {
+        "summary": "Retrieve passholder for user",
```

Comment:
`Retrieve passholder for current user` is probably clearer. The current wording can be interpreted as retrieving the passholder for a given user id for example.